README.md

```diff
@@ -47,10 +47,11 @@ The project is under heavy development. It is stable enough so we use it in prod
 
 A resolver must have one method for each field of the GraphQL type it resolves. The method name has to be [exported](https://golang.org/ref/spec#Exported_identifiers) and match the field's name in a non-case-sensitive way.
 
-The method has up to two arguments:
+The method has up to three arguments:
 
 - Optional `context.Context` argument.
 - Mandatory `*struct { ... }` argument if the corresponding GraphQL field has arguments. The names of the struct fields have to be [exported](https://golang.org/ref/spec#Exported_identifiers) and have to match the names of the GraphQL arguments in a non-case-sensitive way.
+- Optional `[]selected.SelectedField` argument to receive the tree of selected subfields in the GraphQL query (useful for preloading of database relations)
```

Comment:
I don't like the repetition of the work "selected" in: `selected.SelectedField`. It'd be nice if we can avoid that.

Reply:
Maybe `selected.Field` 🤔

Reply:
Ideally, even better package name...

Reply:
Just came across this PR. Given the author has not replied, I'm happy to do these changes (along with tests) if there's any chance that this gets approved and merged afterwards cc/ @pavelnikolov

Reply:
@jesushernandez this is the most requested feature and I really want it to get some traction but I'm not sure that this is the best approach.

Reply:
How can I help getting more traction?